src/components/NcActionInput/NcActionInput.vue

```diff
@@ -540,7 +540,7 @@ $input-margin: 4px;
 		min-height: 0;
 		/* Keep padding to define the width to
 			assure correct position of a possible text */
-		padding: #{math.div($clickable-area, 2)} 0 #{math.div($clickable-area, 2)} $clickable-area;
+		padding: calc(var(--default-clickable-area)/ 2) 0 calc(var(--default-clickable-area)/ 2) var(--default-clickable-area);
```

Comment:
Maybe we should cache `calc(var(--default-clickable-area)/ 2)`? (note for the future)